lib/db/cms/gallery-page.ts

```diff
@@ -1,22 +1,65 @@
-import { collection, getDocs } from 'firebase/firestore'
-import { db } from '../config'
+import { useSelector } from 'react-redux'
 import { GalleryProps } from '@/utils/types/redux'
+import axios from 'axios'
+import { RootState } from '@/lib/redux/store'
 
-const getGalleryContent = async (): Promise<GalleryProps[]> => {
+// Instagrams API currently only gives 25 images at a time
+// It then provoides you with a link for the next api call if there is more then 25 iamges
+
+const getGalleryContent = async (): Promise<GalleryProps> => {
+    const returnData = {
+        nextUrl: '',
+        images: [
+            {
+                media_url: '',
+                caption: ''
+            }
+        ]
+    }
+    const url = `https://graph.instagram.com/me/media?fields=id,media_url,timestamp,caption,permalink&access_token=${process.env.NEXT_PUBLIC_IG_SECRET}`
     try {
-        const galleryRef = collection(db, 'gallery')
-        const data = await getDocs(galleryRef)
-        return data.docs.map(doc => {
-            const { img, tag } = doc.data()
-            return {
-                id: doc.id,
-                img,
-                tag
+        const response = await axios.get(url)
+        const data = response.data
+
+        returnData.nextUrl = data.paging.next
+        returnData.images = data.data.map(
+            (item: { media_url: any; caption: any }) => {
```

Comment:
You dont have to have a return if you arent doing any javascripty things. you can just do `=> (` instead of `=> { return (`. The only reason to explicitly write a return is if you need to make variables within the scope of the map